### ISSUE TITLE
fix(ci): view is horizontally reversed

### ISSUE THREAD

**Issue by davinkevin** [zaquestion/lab#471]
Following a recent upgrade of GitLab (13.5 I think), the result of the `lab ci view` command is totally reversed, on **right to left** axis. 

![image](https://user-images.githubusercontent.com/1970922/97146877-e58a0f00-1768-11eb-9b18-4bfa51923c2f.png)

![image](https://user-images.githubusercontent.com/1970922/97146927-f8044880-1768-11eb-9006-3a313d40a287.png)

Last time I use it was on friday, and today all is **right to left**. 

I didn't change anything to my configuration (I didn't turn the computer on during the week end), and this happen in two different GitLab Instances (gitlab.com and a private managed one).

/cc @neonox31

**Comment by zaquestion** [comment]
@davinkevin i believe this is fixed in main, i remember a test breaking... ill prioritize getting a release out after halloween, im moving atm and my whole life is packed up 😅

**Comment by davinkevin** [comment]
No problem, it's a bit strange to have this behavior but I will wait until you unpack everything.

BTW, moving IRL and from GitHub to GitLab in parallel is funny 😇

**Comment by galan** [comment]
Also affected. Waiting for release since https://github.com/zaquestion/lab/issues/410 . None of the checkboxes is checked since then on https://github.com/zaquestion/lab/issues/406 .

Would be creating a last intermediate release an option to fix this and have the latest changes available (that would fit your packed timetable)? Thanks for the great tool.

**Comment by davinkevin** [comment]
For your information @galan , this is pretty simple to build a version for your OS. Checkout the code, modify the `MAKEFILE` to include your os/arch and `make install`. 

I've tested it, but the `ci view` is still reversed and it totally crash after launching this command 😓. I'll go back to the previous version…

**Comment by davinkevin** [comment]
BTW, there is still some problem with job relauched… they end up at the end of the graph and don't keep the same position. 

Would you like me to open another issue ? 

/cc @fmuellner && @zaquestion

**Comment by fmuellner** [comment]
I've seen that as well - both before and after the fix, so it's a separate issue.

Not sure there's an easy fix though, I didn't find any API for getting stages in order (or at all), only through jobs.
